clientapi/auth/storage/accounts/sqlite3/storage.go

```diff
@@ -134,13 +159,14 @@ func (d *Database) CreateAccount(
 			return nil, err
 		}
 	}
-	if err := d.profiles.insertProfile(ctx, localpart); err != nil {
+	if err := d.profiles.insertProfile(ctx, txn, localpart); err != nil {
 		if common.IsUniqueConstraintViolationErr(err) {
 			return nil, nil
 		}
 		return nil, err
 	}
-	if err := d.SaveAccountData(ctx, localpart, "", "m.push_rules", `{
+
+	if err := d.accountDatas.insertAccountData(ctx, txn, localpart, "", "m.push.rules", `{
```

Comment:
Looks like a typo here - this should be `m.push_rules`.

Reply:
Done. https://github.com/matrix-org/dendrite/pull/882/commits/42a9000d4dfc1f743384e3d59bd948fa4bb08598